### PR TITLE
[MM-51098] Hide "copy invite link" when selecting guest

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Hides the  "copy invite link" button when selecting guest in the invitation modal

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-51098

#### Related Pull Requests
N/A

#### Screenshots
N/A

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by stevemudie** [comment]
Worked as expected @JulienTant!

Once we have a dev approval I'll run E2E tests on it and then approve after that assuming they look good as well. 😄

File: e2e/cypress/tests/integration/enterprise/guest_accounts/guest_invitation_ui_spec.ts
Comment (julmondragon) [inline e2e/cypress/tests/integration/enterprise/guest_accounts/guest_invitation_ui_spec.ts:231]: nit: `// * Select Guest`

**Review by julmondragon — APPROVED** [review]
LGTM ! Thanks @JulienTant

**Review by M-ZubairAhmed — APPROVED** [review]

File: e2e/cypress/tests/integration/enterprise/guest_accounts/guest_invitation_ui_spec.ts
Comment (JulienTant) [reply e2e/cypress/tests/integration/enterprise/guest_accounts/guest_invitation_ui_spec.ts:231]: For me, it's more an action than an assertion. By that I mean that I value the `click()` much more than I value the `should(...)`

**Comment by JulienTant** [comment]
@stevemudie are we good to merge?

**Review by stevemudie — APPROVED** [review]

**Review by pvev — APPROVED** [review]
LGTM! thanks @JulienTant :shipit:

**Comment by JulienTant** [comment]
migrated to https://github.com/mattermost/mattermost-server/pull/22745

**Pull request closed without merging**
